src/utils/getLinkFromText.tsx

```diff
@@ -19,6 +19,6 @@ export const getLinkFromText = (text: string) => {
           {word}{" "}
         </a>
       );
-    else return <span key={index}>{word} </span>;
+    else return <React.Fragment key={index}>{word} </React.Fragment>;
```

Comment:
Do we want to have `<React.Fragment>` or plain `<></>` is enough?

Reply:
@mike-lvov I believe `React.Fragment` is required here, since we're using an array, and it's the only fragment form that supports the `key` prop:

- https://reactjs.org/docs/fragments.html#keyed-fragments